libvips/foreign/webpsave.c

```diff
@@ -839,6 +844,13 @@ vips_foreign_save_webp_class_init(VipsForeignSaveWebpClass *class)
 		G_STRUCT_OFFSET(VipsForeignSaveWebp, lossless),
 		FALSE);
 
+	VIPS_ARG_BOOL(class, "exact", 12,
```

Comment:
The numbers are supposed to be unique -- could you make this 24, or (if you can be bothered) renumber the args?